client/components/index.js

```diff
@@ -47,6 +47,7 @@ class Routes extends Component {
                 <Route path="/admin/pending" exact component={PendingOrders} />
                 <Route path="/checkout" exact component={CheckoutForm} />
                 <Route path="/successfulCheckout" exact component={PaymentSuccess} />
+                <Redirect from='/login' to='/home' />
```

Comment:
Do we need this line, since there are already Redirect lines in the other components? I tested with this line commented out and everything still worked.